### PR TITLE
Set 1ms timeout between keys

### PR DESCRIPTION
**Pull request by Stojdza**
Trying one more time with the timeout between keys. 

I set 1ms timeout for all browsers.

### CODE REVIEW

File: lib/driver-helper.js [hunk 1]
```diff
@@ -207,7 +207,7 @@ export async function setWhenSettable(
 	driver,
 	selector,
 	value,
-	{ secureValue = false, pauseBetweenKeysMS = 0 } = {}
+	{ secureValue = false, pauseBetweenKeysMS = 1 } = {}
```
Comment (alisterscott) [inline lib/driver-helper.js]: I think we need to keep this at 0 _only_ for IE

**Comment by alisterscott** [comment]
See https://github.com/Automattic/wp-calypso/pull/28083 with matching branch name still fails for IE

File: lib/driver-helper.js (hunk 1 above)
Comment (Stojdza) [reply lib/driver-helper.js]: @alisterscott Yeah, it looks like that. But [this](https://github.com/Automattic/wp-e2e-tests/blob/master/lib/components/secure-payment-component.js#L35) line confused me, it sets to 1ms if it's IE?

**Comment by Stojdza** [comment]
Thanks @alisterscott! Looks good for now

**Pull request merged**